/p23_lifecycle_manager:
  ros__parameters:
    managing_node_list: ["acquisition_left", "acquisition_right", "acquisition_center",
     "inference", "slam", "velocity_estimation", "saltas"]
  